deploy/ephemeral.yaml

```diff
@@ -1,8 +1,3 @@
-# A separate Clowder deployment file is required for the project in order to
```

Comment:
I'd like to keep this comment, I guess it'll need to be manually added back if the script removes it.

Reply:
did a little digging and it looks like this package will preserve yaml comments: https://github.com/eemeli/yaml

Reply:
Thanks, I'll check it out tomorrow

Reply:
Swap worked! I re-ran the script and it preserves the comment now